torch_xla/csrc/aten_cpu_fallback.cpp

```diff
@@ -15,11 +15,24 @@
 #include "torch_xla/csrc/function_call_tracker.h"
 #include "torch_xla/csrc/runtime/debug_macros.h"
 #include "torch_xla/csrc/runtime/metrics.h"
+#include "torch_xla/csrc/runtime/runtime.h"
 #include "torch_xla/csrc/runtime/tf_logging.h"
 #include "torch_xla/csrc/xla_graph_executor.h"
 
 namespace torch_xla {
 
+// List of operations that should be fallbacked to CPU instead of GPU.
+static std::unordered_set<std::string> _force_fallback_on_cpu{
+    // This operation is a simple memory access that transforms the given
+    // 1-element tensor into a Scalar.
+    //
+    // Although it makes sense to run this operation on CPU (since the
+    // output will get copied back to CPU anyway), this also fixes a
+    // particular issue with moco benchmark.
+    // More details: https://github.com/pytorch/xla/issues/7647
+    "aten::_local_scalar_dense",
+};
```

Comment:
Just to be completely transparent: `aten::_local_scalar_dense` is here for 2 reasons:
1. It just makes sense to run it on CPU, since the output (a `Scalar`) also lives on CPU
2. As a temporary fix to #7647